src/PKSim.Core/Commands/EditIndividualMoleculeExpressionInSimulationSubjectFromQueryCommand.cs

```diff
@@ -37,7 +37,7 @@ public override void Execute(IExecutionContext context)
             if (expressionParameter.Value == expressionResult.RelativeExpression)
                continue;
 
-            Add(new SetRelativeExpressionCommand(expressionParameter, expressionResult.RelativeExpression, updateIndividuals:false));
+            Add(new SetExpressionProfileValueCommand(expressionParameter, expressionResult.RelativeExpression, updateSimulationSubjects:false));
```

Comment:
`SetExpressionProfileValueCommand` that is triggered when an expression profile parameter value is changed (Global parameter or local parameter). it's not possible to change anything else in the simulation (e.g. transport direction etc..)